models/staging/tpdm/base/base_tpdm__evaluation_element.sql

```diff
@@ -0,0 +1,26 @@
+with educator_prep_programs as (
+    {{ source_edfi3('evaluation_element') }}
+),
+renamed as (
+    select
+        tenant_code,
+        api_year,
+        pull_timestamp,
+        last_modified_timestamp,
+        file_row_number,
+        filename,
+        is_deleted,
+
+        v:evaluationElementTitle::string as evaluation_element_title,
+        v:sortOrder::int                 as sort_order,
+        v:minRating::decimal             as min_rating,
+        v:maxRating::decimal             as max_rating,
```

Comment:
I'm not sure about type, what type here should be? it is DECIMAL(3,6) in docs 
![image](https://github.com/user-attachments/assets/317536c0-8d4f-4797-a2a8-0e55d9906944)